### PR TITLE
Changelog

### PR DESCRIPTION
**Pull request by kacf**

### CODE REVIEW

**Comment by kacf** [comment]
@michaelatmender: Can you check this as well? :-)

Should be cherry-picked to 1.0.x.

File: 202.Release-information/01.Release-notes-changelog/docs.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  artifact description
+* Move user token from local storage to cookie, add react-cookie module (#217)
+* Update node modules, add drag+drop & cookie functionality (#219)
+* Replace artifact upload dialog with drag-and-drop
+* Remove cookie when receiving unauthorized response
+* Edit artifact description in UI
+
+#### mender
+
+* Fix bug that caused the update to be retried only once.
```
Comment (pasinskim) [inline 202.Release-information/01.Release-notes-changelog/docs.md]: Maybe `Fix bug that caused the update was not retried after failing during previous attempt (#193)`

File: 202.Release-information/01.Release-notes-changelog/docs.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+## v1.0.1 
+_Released dd.mm.yyyy_
+
+### Notable changes
+
+#### deployments
+
+* Update artifact description when updating artifact data. (MEN-1093)
+* Fix log flag not being set for device deployment after log been uploaded.
+	  (MEN-1078)
```
Comment (pasinskim) [inline 202.Release-information/01.Release-notes-changelog/docs.md]: The formatting here seems to be quite bad.

File: 202.Release-information/01.Release-notes-changelog/docs.md (hunk 1 above)
Comment (kacf) [inline 202.Release-information/01.Release-notes-changelog/docs.md]: Fixed!

File: 202.Release-information/01.Release-notes-changelog/docs.md (hunk 2 above)
Comment (kacf) [inline 202.Release-information/01.Release-notes-changelog/docs.md]: Fixed!

File: 202.Release-information/01.Release-notes-changelog/docs.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+#### gui
```
Comment (pasinskim) [inline 202.Release-information/01.Release-notes-changelog/docs.md]: This is just general comment. We need to work a bit on unifying those changelog messages.

Comment (kacf) [inline 202.Release-information/01.Release-notes-changelog/docs.md]: Indeed. My task on adapting the changelog generator has not even started yet! This was just a quick hack with a lot of manual work thrown on top.

**Comment by kacf** [comment]
Cherry-picked to 1.0.x as well.

**Pull request merged**